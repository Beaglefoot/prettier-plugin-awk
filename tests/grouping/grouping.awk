{
  x = 5 / (  1+2)
}
