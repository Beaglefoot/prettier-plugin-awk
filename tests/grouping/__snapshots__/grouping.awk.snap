{
  x = 5 / (  1+2)
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ x = 5 / (1 + 2) }
